templates/gallery.tmpl

```diff
@@ -0,0 +1,27 @@
+{{ define "head" }}
+    <title>Gallery | HackSoc</title>
+    <link rel="stylesheet" type="text/css" href="assets/css/gallery.min.css" />
+    <script src="../assets/javascript/jquery.min.js"></script>
+{{ end }}
+
+{{ define "content" }}
+    <section id="gallery">
+        <h2>Gallery</h2>
+        <div class="gallery-container">
+            {{ range .Photosets.Photoset }}
+                <a href="http://www.flickr.com/photos/147757828@N02/sets/{{ .ID }}" target="_blank">
+                    <div class="image-container" style='background-image: url("https://farm{{ .Farm }}.staticflickr.com/{{ .Server }}/{{ .Primary }}_{{ .Secret }}.jpg")'>
```

Comment:
oh, it cannot... ok... I still have an idea on how to fix the other issues.
about the secret... I can try to find a workaround but, just so you know, if you go on flickr and you open any photo you can actually inspect element and see an img tag with a link just like this one, with secret and everything...